make/autoconf/build-performance.m4

```diff
@@ -23,6 +23,10 @@
 # questions.
 #
 
+# ===========================================================================
+# (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+# ===========================================================================
+
```

Comment:
Please don't add copyright notices like this, for file which have no IBM content (this file is modified only by reverting some upstream changes). None of the files in `make/autoconf` in this change should have added or updated IBM copyright notices.